manim/animation/rotation.py

```diff
@@ -3,6 +3,7 @@
 __all__ = ["Rotating", "Rotate"]
 
 import typing
```

Comment:
is this good practice to have these imports typing and from typing import Callable, Optional, Sequence ?

Reply:
It's just namespacing... but since we're using it directly to shorten it, in this case it's an unused import and can be removed.